### PR TITLE
Add explanation to Part 1 introduction

### LINKED ISSUE #90: [JavaFX] Introduction in Part 1 can be clearer to beginners
Introduction in Part 1 introduced Stage, Scene, Node and Root node structure without going into details. Further explanation can be added to reinforce understanding.

### PR DESCRIPTION
**Pull request by laney0808**
Explain the meaning of Stage, Scene, Root node and Node in Part 1 introduction for clarity.
Fixes #90

### CODE REVIEW

**Comment by laney0808** [comment]
Fix #90

**Comment by damithc** [comment]
> Fix #90

@laney0808 This should be in the PR description (not a separate comment), and should be `Fixes #90`

File: tutorials/javaFxPart1.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ Imagine yourself as a director of a play. First you provision the props that you
 
 A JavaFX application is like a play you are directing. Instead of creating props, you create `Node`s (`Node`s are the fundamental building blocks of a JavaFX application), and place them onto a `Scene` (a scene is a graph of `Node`s). Then, you set your `Scene` on a `Stage` provided by JavaFX. When you call `Stage#show()` method, JavaFX renders a window with your `Stage` on it.
 
+<box type="tip" seamless>
+The `Stage` is like a window in a desktop application. It is the top-level container for a JavaFX application.
```
Comment (gongg21) [inline tutorials/javaFxPart1.md:19]: Maybe we could add, for greater clarity, that the Stage consists of different scenes at different times, i.e. depending on the current action/state, the Scene on the Stage is different?

Comment (laney0808) [reply tutorials/javaFxPart1.md:19]: Great point! Will add in the next commit. Thank you!

**Comment by damithc** [comment]
Thanks for this PR @laney0808 
I've adapted its content in https://github.com/se-edu/guides/commit/c7a42cf1376d074c5d043275be4b82d24da45342

In future, remember to preview the changes locally first. The code in this PR doesn't render as intended because it doesn't have a blank line below the `<box>` tag. Local previewing can catch errors like that one.

**Pull request closed without merging**